taskcluster/ci/config.yml

```diff
@@ -6,8 +6,10 @@ task-priority: lowest
 taskgraph:
     register: xpi_taskgraph:register
     repositories:
-        xpi:
-            name: "XPI source"
```

Comment:
You should not remove the `xpi` entry, as that will populate the right env variables when checking things out (though you will need to adjust the `checkout:` definitions in tasks to specify which repositories you want to checkout.

Reply:
We may need to adjust how taskgraph handles multiple repositories. I only really had one example when writing it, and it had a slightly different shape to it.

Reply:
I think this is now resolved, with your help. Thanks!